src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -537,4 +538,17 @@ public String getName() {
 
 	protected abstract Endpoint getEndpoint();
 
+	protected boolean isSuperAdmin() {
+		User user = threadPool.getThreadContext().getTransient(ConfigConstants.OPENDISTRO_SECURITY_USER);
+		return adminDNs.isAdmin(user);
+	}
+
+	protected boolean isReservedAndAccessibleToSuperAdmin(final SecurityDynamicConfiguration<?> existingConfiguration,
```

Comment:
PR looks good to me. 
Not sure if there is a better name for `isReservedAndAccessibleToSuperAdmin` or maybe use `isReserved` itself and document that super admins can update reserved resources.

Reply:
What are your thoughts?

Reply:
I think rename is required. From what I think, isReservedAndAccessible should be the one. Because it will make more sense. isReserved we can do but I think it wont go with the code.
Your call - isReserved or isReservedAndAccessible?

Reply:
Are there any resources which are reserved and not accessible to super-admins ?

Reply:
We are allowing actiongroups, internalusers, roles, rolesmapping, tenants.
Account & Permission is not allowed to edit. 
Are they also required?